Integrations/AWS-EC2/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-Improved handling of error messages.
```

Comment:
was this intended?
why do you remove RN?

Reply:
Ah no, not intended... I'll fix